hoomd/md/methods/methods.py

```diff
@@ -327,29 +372,79 @@ class ConstantPressure(Thermostatted):
             the barostat.
             In Voigt notation,
             :math:`[S_{xx}, S_{yy}, S_{zz}, S_{yz}, S_{xz}, S_{xy}]`
-            :math:`[\mathrm{pressure}]`. Stress can be reset after the method
-            object is created. For example, an isotropic pressure can be set by
-            ``npt.S = 4.``
+            :math:`[\mathrm{pressure}]`.
+
+            .. rubric:: Examples:
+
+            .. code-block:: python
+
+                npt.S = 4.0
+
+            .. code-block:: python
+
+                npt.S = hoomd.variant.Ramp(A=1.0,
+                                           B=2.0,
+                                           t_start=0,
+                                           t_ramp=1_000_000)
 
         tauS (float): Coupling constant for the barostat
             :math:`[\mathrm{time}]`.
 
+            .. rubric:: Example:
+
+            .. code-block:: python
+
+                npt.tauS = 2.0
+
         couple (str): Couplings of diagonal elements of the stress tensor,
-            can be "none", "xy", "xz","yz", or "xyz".
+            can be 'none', 'xy', 'xz', 'yz', or 'xyz'.
+
+            .. rubric:: Example:
+
+            .. code-block:: python
+
+                npt.couple = 'none'
+
+        box_dof(list[bool]): Box degrees of freedom with six boolean elements in
+            the order [x, y, z, xy, xz, yz].
+
+            .. rubric:: Example:
+
+            .. code-block:: python
+
+                npt.box_dof = [False, False, True, False, False, False]
 
-        box_dof(list[bool]): Box degrees of freedom with six boolean elements
-            corresponding to x, y, z, xy, xz, yz, each.
+        rescale_all (bool): When True, rescale all particles, not only those
+            selected by the filter.
 
-        rescale_all (bool): if True, rescale all particles, not only those in
-            the group.
+            .. rubric:: Example:
 
-        gamma (float): Friction constant for the box degrees of freedom, Default
-            to 0  :math:`[\mathrm{time^{-1}}]`.
+            .. code-block:: python
+
+                npt.rescale_all = True
+
+        gamma (float): Friction constant for the box degrees of freedom
+            :math:`[\mathrm{time^{-1}}]`.
 
         barostat_dof (tuple[float, float, float, float, float, float]):
             Additional degrees of freedom for the barostat (:math:`\nu_{xx}`,
             :math:`\nu_{xy}`, :math:`\nu_{xz}`, :math:`\nu_{yy}`,
             :math:`\nu_{yz}`, :math:`\nu_{zz}`)
+
+            Save and restore the barostat degrees of freedom when continuing
+            simulations:
+
+            .. rubric:: Examples:
+
+            .. code-block:: python
+
+                saved_barostat_dof = npt.barostat_dof
+                # Save to a file.
+
+            .. code-block:: python
+
+                # Load from the file on next execution.
```

Comment:
Again, same as above.
```suggestion
                # Load from the file on next execution.
                import numpy
                saved_barostat_dof = numpy.loadtxt('barostat_dof.txt')
```